SIGs/GRC/publications/Verifier_Governance.md

```diff
@@ -34,7 +34,7 @@ The list below assumes a multi-tenant Verifier (single-tenant is a trivial case
 | **Verifier Service software** | VS | Deployed executable code that constitutes the Verifier Service and is shared by all Verifier Tenants | No |
 | **Verifier Service configuration** | VS | Configuration values that define the execution of the Verifier Service and integration with its supporting services (if any), *excluding* cryptographic keys | No |
 | **Verifier Service secrets** | VS | Secrets (encryption, signing) utilized by the Verifier Service for its secure operation | Yes |
-| **Verifier Service Reference Values** | VS | Expected Reference Values used in the process of assessing Evidence produced by an Attester. Informally these are the hardware reference values. | No |
+| **Verifier Service Reference Values** | VS | Expected Reference Values used in the process of assessing Evidence produced by an Attester. Informally these are the hardware and firmware reference values. | No |
 | **Verifier Tenant secrets** | VT |  All Verifier Tenant secrets for authentication or encryption | Yes |
 | **Verifier Tenant signing key(s)** | VT | The signing key (or similar) used to sign the Verifier Tenant responses such as Attestation Results | Yes |
 | **Verifier Tenant encryption key(s)** | VT | The encryption keys used to safeguard Verifier Tenant data at rest | Yes |
```

Comment:
You don't say anything about Verifier Tenant reference values. Now that we have specified what the VS reference values are, we should do the same for VT, and it's fine to use the "predominant operational model" language here.

Reply:
Actually, "Verifier Service Reference Values" should probably be "TEE Reference Values"
And then we can add "Workload Reference Values" to address your feedback on doing the same for the VT.
If you agree I can push up an edit.

Reply:
The intent was to say that Verifier maintains Reference Values for the customer, and yes, those reference values refer to the customer's TEEs, but they are a Verifier asset. Not sure if we simply need to word this better...